client/core/core.go

```diff
@@ -49,6 +49,8 @@ type dexConnection struct {
 	acct        *dexAccount
 	booksMtx    sync.RWMutex
 	books       map[string]*book.OrderBook
+	epochsMtx   sync.RWMutex
+	epochs      map[string]*book.EpochQueue
```

Comment:
I think the `OrderBook` should handle the `EpochQueue`, either wrapping it's methods or exposing the queue as an exported field. The queue would be created in the `OrderBook` constructor.